mamba_ssm/models/mixer_seq_simple.py

```diff
@@ -234,8 +338,8 @@ def forward(self, input_ids, position_ids=None, inference_params=None, num_last_
         if num_last_tokens > 0:
             hidden_states = hidden_states[:, -num_last_tokens:]
         lm_logits = self.lm_head(hidden_states)
-        CausalLMOutput = namedtuple("CausalLMOutput", ["logits"])
-        return CausalLMOutput(logits=lm_logits)
+        LMOutput = namedtuple("LMOutput", ["logits"])
```

Comment:
I know this isn't from your code, but constructing named tuples in this way is deprecated as it prevents proper type inference.